archive/DOCKERFILE

```diff
@@ -0,0 +1,13 @@
+FROM public.ecr.aws/lambda/python:latest
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY db_connection.py .
+COPY clean_national_rail.py .
+COPY clean_real_time_trains.py .
+COPY archive.py .
+
+CMD [ "archive.handler" ]
+
+# docker build --platform "linux/amd64" -t c11-railway-tracker-archive-lambda-ecr .
```

Comment:
This comment should not be part of the final build, as this is just a personal note

Reply:
agreed!